src/modules/user/entities/user.entity.ts

```diff
@@ -11,7 +11,11 @@ import {
   ManyToOne,
   RelationId,
 } from 'typeorm';
-import { RoleEnum } from '../../auth/role.enum';
+
+export enum UserTypeEnum {
+  Vendor = 'vendor',
+  Costumer = 'costumer'
```

Comment:
```suggestion
  Customer = 'customer'
```
в нас роль покупця а не костюмера ))